hystrix/src/test/java/feign/hystrix/HystrixBuilderTest.java

```diff
@@ -148,7 +148,7 @@ public void fallbacksApplyOnError() {
   @Test
   public void errorInFallbackHasExpectedBehavior() {
     thrown.expect(HystrixRuntimeException.class);
-    thrown.expectMessage("contributors failed and fallback failed.");
+    thrown.expectMessage("feign#public abstract java.util.List feign.hystrix.HystrixBuilderTest$GitHub.contributors(java.lang.String,java.lang.String) failed and fallback failed.");
```

Comment:
Guess I am not sure what you mean by without any gain in uniqueness?  Right now the key is the method name now the key is the entire signature which makes it much more unique.  

What is the convention you are referring to?